test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -73,6 +73,51 @@
      [9 "2nd sunday"     #t "2000-01-09"]
      [10 "2005 saturday" #t "2005-01-01"]]]])
 
+(mt/defdataset times-mixed-1
```

Comment:
duplicate this so that when run tests CI will create a new version of this dataset that include the `dt_tz` column for bigquery and snowflake.

Once this PR is approved I'll update the original `times-mixed` dataset and remove this one.